src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/UpdatePackageCommand.cs

```diff
@@ -158,7 +158,20 @@ protected override void WarnIfParametersAreNotSupported()
                 var projectNames = string.Join(",", Projects.Where(e => e is BuildIntegratedNuGetProject).Select(p => NuGetProject.GetUniqueNameOrName(p)));
                 if (!string.IsNullOrEmpty(projectNames))
                 {
-                    var warning = string.Format(CultureInfo.CurrentUICulture, Resources.Warning_SourceNotRespectedForProjectType, nameof(Source), projectNames);
+                    var warning = string.Format(CultureInfo.CurrentCulture, Resources.Warning_SourceNotRespectedForProjectType, nameof(Source), projectNames);
+                    Log(MessageLevel.Warning, warning);
+                }
+            }
+        }
+
+        private void WarnForReinstallOfBuildIntegratedProjects(IEnumerable<BuildIntegratedNuGetProject> projects)
+        {
+            if (projects.Any())
+            {
+                var projectNames = string.Join(",", projects.Select(p => NuGetProject.GetUniqueNameOrName(p)));
+                if (!string.IsNullOrEmpty(projectNames))
```

Comment:
Why would this be null/empty? If it is, I'd say still warn anyway and just omit the project names. Otherwise the user may expect something happened, but it did not.

Or... it's impossible for this to be null, so remove the check.

Reply:
Can't be null. I removed the check.